01 - Navigation/01 - Navigate to something you know/Code/JetBrains/Customers/Customer.php

```diff
@@ -57,4 +57,4 @@ public function getName()
     public function celebrateBirthday() {
         echo 'Yay!';
     }
-} 
\ No newline at end of file
+}
```

Comment:
I didn't look into the other/existing files, just looked at the changes in this PR. Although I agree with being "consistent", I'm not sure introducing (what I see as) errors to be consistent, is the best approach.

Please, don't see this as a personal attack, I really appreciate your efforts. I try to be honest when reviewing code. Also, I'm not affiliated with JetBrains, so I don't have *any* say in this.